pyuvdata/cst_beam.py

```diff
@@ -197,11 +197,8 @@ def read_cst_beam(self, filename, beam_type='power', feed_pol='x',
             if not np.allclose(roll_rot_phi, phi_data):
                 raise ValueError('Rotating by pi/2 failed')
 
-            # theta should not be affected by the rotation
+            # theta is not affected by the rotation
             rot_theta = theta_data
-            roll_rot_theta = np.roll(rot_theta, int((np.pi / 2) / delta_phi), axis=1)
-            if not np.allclose(roll_rot_theta, theta_data):
-                raise ValueError('Rotating by pi/2 failed')
```

Comment:
I made this change because I found that it was impossible to trigger this error, by construction.